### PR TITLE
Added changelog

### PR DESCRIPTION
**Pull request by tillkruss**
Make it easier to keep track of changes in the `laravel/laravel` repository.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## v5.4.16 (2017-03-17)
+
+### Added
+- Added `unix_socket` to `mysql` in `config/database.php` ()[#4179](https://github.com/laravel/laravel/pull/4179))
+- Added Pusher example code to `bootstrap.js` ([31c2623](https://github.com/laravel/laravel/commit/31c262301899b6cd1a4ce2631ad0e313b444b131))
+
+### Changed
+- Use `smtp.mailtrap.io` as default `MAIL_HOST` ([#4182](https://github.com/laravel/laravel/pull/4182))
+- Upgrade Laravel Mix to `0.8.3` ([#4174](https://github.com/laravel/laravel/pull/4174))
```
Comment (GrahamCampbell) [inline CHANGELOG.md]: 👎 That PR didn't do anything anyway.

Comment (tillkruss) [inline CHANGELOG.md]: Right, thanks. I'll omit thing that don't do anything.

Comment (GrahamCampbell) [inline CHANGELOG.md]: :D

**Comment by GrahamCampbell** [comment]
I'm not sure it makes sense to have a change log in what is actually, a default application, since this doesn't represent the change log for somebodies app.

**Comment by taylorotwell** [comment]
One thing I could do is remove this from applications created via `laravel new`.

**Comment by tillkruss** [comment]
@taylorotwell: I just added `CHANGELOG.md` to the `export-ignore` list, that will do the trick AFAIK.

**Pull request merged**